.github/labeler.yaml

```diff
@@ -0,0 +1,5 @@
+# Configuration for .github/workflows/pull_request_label.yml.
```

Comment:
Not for this PR, but we'll also need a `workflows/pull_request_labeller.yml` file for this config to take effect.

Reply:
https://github.com/dart-lang/labs/blob/main/.github/workflows/pull_request_label.yml or similar.